tests/unit/dataactbroker/test_account_handler.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import Mock
+import dataactbroker.handlers.accountHandler
+import json
+from dataactcore.utils.jsonResponse import JsonResponse
+from dataactcore.utils.statusCode import StatusCode
+
+
+def test_max_login_success(database, user_constants, monkeypatch):
+    ah = dataactbroker.handlers.accountHandler.AccountHandler(Mock())
+
+    mock_dict = Mock()
+    mock_dict.return_value.safeDictionary.side_effect = {'ticket': '', 'service': ''}
+    monkeypatch.setattr(dataactbroker.handlers.accountHandler, 'RequestDictionary', mock_dict)
+
+    max_dict= {'cas:serviceResponse': {}}
+    monkeypatch.setattr(ah, 'get_max_dict', Mock(return_value=max_dict))
+    config = {'parent_group': 'parent-group'}
+    monkeypatch.setattr(dataactbroker.handlers.accountHandler, 'CONFIG_BROKER', config)
+    max_dict = {
```

Comment:
Do you think `max_dict` is a candidate for a fixture that can be updated as needed for the individual scenarios? Or is that overkill since it's a resource that will be used in a very limited number of tests?